2025-02-11 00:41:25,309 INFO    MainThread:2229527 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2025-02-11 00:41:25,310 INFO    MainThread:2229527 [wandb_setup.py:_flush():76] Configure stats pid to 2229527
2025-02-11 00:41:25,310 INFO    MainThread:2229527 [wandb_setup.py:_flush():76] Loading settings from /common/home/hg343/.config/wandb/settings
2025-02-11 00:41:25,311 INFO    MainThread:2229527 [wandb_setup.py:_flush():76] Loading settings from /common/home/hg343/Research/accelerate_combo_option/src/wandb/settings
2025-02-11 00:41:25,311 INFO    MainThread:2229527 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'api_key': '***REDACTED***'}
2025-02-11 00:41:25,311 INFO    MainThread:2229527 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'src/combo_stock_frontier_data_preprocessor.py', 'program_abspath': '/common/home/hg343/Research/accelerate_combo_option/src/combo_stock_frontier_data_preprocessor.py', 'program': '/common/home/hg343/Research/accelerate_combo_option/src/combo_stock_frontier_data_preprocessor.py'}
2025-02-11 00:41:25,312 INFO    MainThread:2229527 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-02-11 00:41:25,312 INFO    MainThread:2229527 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-02-11 00:41:25,313 INFO    MainThread:2229527 [wandb_init.py:_log_setup():520] Logging user logs to /common/home/hg343/Research/accelerate_combo_option/src/wandb/run-20250211_004125-j9jpgk3l/logs/debug.log
2025-02-11 00:41:25,313 INFO    MainThread:2229527 [wandb_init.py:_log_setup():521] Logging internal logs to /common/home/hg343/Research/accelerate_combo_option/src/wandb/run-20250211_004125-j9jpgk3l/logs/debug-internal.log
2025-02-11 00:41:25,313 INFO    MainThread:2229527 [wandb_init.py:init():560] calling init triggers
2025-02-11 00:41:25,313 INFO    MainThread:2229527 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2025-02-11 00:41:25,314 INFO    MainThread:2229527 [wandb_init.py:init():610] starting backend
2025-02-11 00:41:25,314 INFO    MainThread:2229527 [wandb_init.py:init():614] setting up manager
2025-02-11 00:41:25,315 INFO    MainThread:2229527 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-02-11 00:41:25,317 INFO    MainThread:2229527 [wandb_init.py:init():622] backend started and connected
2025-02-11 00:41:25,321 INFO    MainThread:2229527 [wandb_init.py:init():711] updated telemetry
2025-02-11 00:41:25,341 INFO    MainThread:2229527 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2025-02-11 00:41:25,544 INFO    MainThread:2229527 [wandb_run.py:_on_init():2396] communicating current version
2025-02-11 00:41:25,583 INFO    MainThread:2229527 [wandb_run.py:_on_init():2405] got version response upgrade_message: "wandb version 0.19.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-02-11 00:41:25,584 INFO    MainThread:2229527 [wandb_init.py:init():795] starting run threads in backend
2025-02-11 00:41:38,661 INFO    MainThread:2229527 [wandb_run.py:_console_start():2374] atexit reg
2025-02-11 00:41:38,662 INFO    MainThread:2229527 [wandb_run.py:_redirect():2229] redirect: wrap_raw
2025-02-11 00:41:38,662 INFO    MainThread:2229527 [wandb_run.py:_redirect():2294] Wrapping output streams.
2025-02-11 00:41:38,662 INFO    MainThread:2229527 [wandb_run.py:_redirect():2319] Redirects installed.
2025-02-11 00:41:38,664 INFO    MainThread:2229527 [wandb_init.py:init():838] run started, returning control to user process
2025-02-11 00:42:09,465 INFO    MainThread:2229527 [wandb_run.py:_finish():2103] finishing run jglppk/expediating_comb_financial_market_matching/j9jpgk3l
2025-02-11 00:42:09,465 INFO    MainThread:2229527 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 1
2025-02-11 00:42:09,465 INFO    MainThread:2229527 [wandb_run.py:_restore():2326] restore
2025-02-11 00:42:09,466 INFO    MainThread:2229527 [wandb_run.py:_restore():2332] restore done
2025-02-11 00:42:14,294 WARNING MsgRouterThr:2229527 [router.py:message_loop():77] message_loop has been closed
